ToggleButton/ToggleButton.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Agate styled button components and behaviors.
+ *
+ * @example
+ * <Button small>Hello Enact!</Button>
+ *
+ * @module agate/Button
+ * @exports Button
+ * @exports ButtonBase
+ * @exports ButtonDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Pure from '@enact/ui/internal/Pure';
+import Toggleable from '@enact/ui/Toggleable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Button from '../Button';
+// import {MarqueeDecorator} from '../Marquee';
```

Comment:
I'm guessing this is for a future implementation, but it looks like `agate/Button` has the same plans. If it's implemented in `Button`, we could probably remove this. Thoughts?

Reply:
removed!